sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.searchStudents.e2e-spec.ts

```diff
@@ -393,6 +394,44 @@ describe("StudentInstitutionsController(e2e)-searchStudents", () => {
     };
   };
 
+  it("Should throw forbidden error when the institution type is not BC Public.", async () => {
+    // Arrange
+
+    // Student submitting an application to College C.
+    const { collegeCApplication } = await saveStudentApplicationForCollegeC(
```

Comment:
Do we need to create any extra data on DB to validate this scenario?

Reply:
Makes sense. Removing.